Nautilus/Assets/SpawnLocation.cs

```diff
@@ -7,4 +7,5 @@ namespace Nautilus.Assets;
 /// </summary>
 /// <param name="Position">The world position.</param>
 /// <param name="EulerAngles">Euler angles for the rotation the spawned object will appear with.</param>
-public record SpawnLocation(Vector3 Position, Vector3 EulerAngles = default);
\ No newline at end of file
+/// <param name="Scale">The scale that the object is spawned at. If default (0, 0, 0) will be resolved to (1, 1, 1).</param>
+public record SpawnLocation(Vector3 Position, Vector3 EulerAngles = default, Vector3 Scale = default);
```

Comment:
Breaking change!

Keep the previous signature intact, then add a body to the record and add the new scale parameter there.

Something like this:
```suggestion
public record SpawnLocation(Vector3 Position, Vector3 EulerAngles = default)
{
    public Vector3 Scale { get; init; }
    
    /// <summary>
    /// Defines the spawn location with world position and optional euler angles. Used in the Coordinated Spawns system.
    /// </summary>
    /// <param name="Position">The world position.</param>
    /// <param name="EulerAngles">Euler angles for the rotation the spawned object will appear with.</param>
    /// <param name="Scale">The scale that the object is spawned at. If default (0, 0, 0) will be resolved to (1, 1, 1).</param>
    public SpawnLocation(Vector3 Position, Vector3 EulerAngles = default, Vector3 Scale = default) : this(Position, EulerAngles)
    {
        this.Scale = Scale;
    }
}
```